gslb/cache/controller_obj_cache.go

```diff
@@ -571,10 +575,28 @@ func GetDetailsFromAviGSLB(gslbSvcMap map[string]interface{}) (uint32, []GSMembe
 				weight = 0
 			}
 			weightI := int32(weight)
-			ipList = append(ipList, ipAddr+"-"+strconv.Itoa(int(weightI)))
+			vsUUID, ok := member["vs_uuid"].(string)
+			if !ok {
+				gslbutils.Warnf("couldn't parse the vs uuid, assigning \"\": %v", member)
+				vsUUID = ""
+			}
+			controllerUUID, ok := member["cluster_uuid"].(string)
+			if !ok {
+				gslbutils.Warnf("couldn't parse the controller cluster uuid, assigning \"\": %v", member)
+				controllerUUID = ""
+			}
+			var server string
+			if vsUUID != "" {
+				server = vsUUID + "-" + controllerUUID
```

Comment:
Can the controllerUUID be empty? In that case, will the `server` string be valid?

Reply:
Yes it could be. Changed it in the latest commit where we don't add the controller UUID and the VS UUID in the rest layer if the sync type is vip only.

Reply:
Minor Nit: here while populating cache, we are deriving the server by checking, if vsUUID != "", but while building the model we are checking - if !gsMember.SyncVIPOnly. 
It seems to me that we can have common checks for both and SyncVIPOnly can be avoided fro the gsmemeber in model.